site/docs/tools/iceberg/spark.md

```diff
@@ -95,80 +120,17 @@ auth_type = "NONE"
-    String url = "http://localhost:19120/api/v1";
-    // Where to store nessie tables
-    String fullPathToWarehouse = ...;
-    // The ref or context that nessie will operate on
-    // (if different from default branch).
-    // Can be the name of a Nessie branch or tag name. 
-    String ref = "main";
-    // Nessie authentication type (BASIC, NONE or AWS)
-    String authType = "NONE";
-
-    //for a local spark instance
-    conf.set("spark.jars.packages",
-            "org.apache.iceberg:iceberg-spark-runtime-{{ versions.iceberg_spark32 }}:{{ versions.iceberg }},org.projectnessie:nessie-spark-3.2-extensions:{{ versions.java }}")
-        .set("spark.sql.extensions", 
-            "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions,org.projectnessie.spark.extensions.NessieSpark32SessionExtensions")
-        .set("spark.sql.catalog.nessie.uri", url)
-        .set("spark.sql.catalog.nessie.ref", ref)
-        .set("spark.sql.catalog.nessie.authentication.type", authType)
-        .set("spark.sql.catalog.nessie.catalog-impl",
-            "org.apache.iceberg.nessie.NessieCatalog")
-        .set("spark.sql.catalog.nessie.warehouse", fullPathToWarehouse)
-        .set("spark.sql.catalog.nessie", "org.apache.iceberg.spark.SparkCatalog");
-    spark = SparkSession.builder()
-                        .master("local[2]")
-                        .config(conf)
-                        .getOrCreate();
-    ```
-=== "Python"
-    ``` python
-    # Full url of the Nessie API endpoint to nessie
-    url = "http://localhost:19120/api/v1"
-    # Where to store nessie tables
-    full_path_to_warehouse = ...
-    # The ref or context that nessie will operate on (if different from default branch).
-    # Can be the name of a Nessie branch or tag name.
-    ref = "main"
-    # Nessie authentication type (BASIC, NONE or AWS)
-    auth_type = "NONE"
-
-    # here we are assuming NONE authorisation
-    spark = SparkSession.builder \
-            .config("spark.jars.packages",
-                "org.apache.iceberg:iceberg-spark-runtime-{{ versions.iceberg_spark32 }}:{{ versions.iceberg }},org.projectnessie:nessie-spark-3.2-extensions:{{ versions.java }}") \
-            .config("spark.sql.extensions", 
-                "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions,org.projectnessie.spark.extensions.NessieSparkSessionExtensions") \
-            .config("spark.sql.catalog.nessie.uri", url) \
-            .config("spark.sql.catalog.nessie.ref", ref) \
-            .config("spark.sql.catalog.nessie.authentication.type", auth_type) \
-            .config("spark.sql.catalog.nessie.catalog-impl", 
-                "org.apache.iceberg.nessie.NessieCatalog") \
-            .config("spark.sql.catalog.nessie.warehouse", full_path_to_warehouse) \
-            .config("spark.sql.catalog.nessie",
-                "org.apache.iceberg.spark.SparkCatalog") \
-            .getOrCreate()
-    ```
```

Comment:
this file has a section called `## Writing` and it has spark2.4 mentioning and spark2 section. Maybe we should remove it too. 

line 153 in the latest version of the file.